### PR TITLE
split up listing of libraries between openzipkin and community supported

### PR DESCRIPTION
**Pull request by basvanbeek**
Split up the listing of tracing libraries between OpenZipkin supported and Community supported ones for more clarity on where to go for support.

cc @adriancole I made the font size for the tables smaller as the font is rather big and they've become quite wide. So please check if you are ok with both the content change and the looks too.

### CODE REVIEW

File: _data/openzipkin_instrumentations.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  propagation: Http (B3)
+  transports: >-
+    [Http, Kafka, Scribe](https://github.com/openzipkin/zipkin-js#transports)
+  sampling: >-
+    Yes
+  notes: >-
+    Uses continuation-local-storage under to hood, so you don't have to pass around an explicit context
+
+- language: Ruby
+  library: >-
```
Comment (adriancole) [inline _data/openzipkin_instrumentations.yml]: Renamed repo to zipkin-ruby (not gem)

File: _data/openzipkin_instrumentations.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    [zipkin-tracer](https://github.com/openzipkin/zipkin-tracer)
+  framework: >-
+    [Rack](http://rack.github.io/)
+  propagation: Http (B3)
+  transports: Http, Kafka, Scribe
+  sampling: "Yes"
+  notes: lc support. Ruby 2.0 or higher
+
+- language: Scala
```
Comment (adriancole) [inline _data/openzipkin_instrumentations.yml]: Technically it is java, but more correct to say scala as it has runtime dep

File: pages/existing_instrumentations.md [hunk 3]
```diff
@@ -12,8 +12,20 @@ The following libraries exist to provide instrumentation on various platforms.
 Please refer to their individual documentation for setup and configuration
 guides.
 
+### openzipkin libraries
+
+The following libraries are supported by the OpenZipkin authors.
```
Comment (adriancole) [inline pages/existing_instrumentations.md]: Maybe note where to find support (ex on gitter)

**Comment by adriancole** [comment]
cc @openzipkin/core @mjbryant @shakuzen @nicmunroe

**Comment by mjbryant** [comment]
👍 this is helpful

File: _data/openzipkin_instrumentations.yml (hunk 2 above)
Comment (basvanbeek) [inline _data/openzipkin_instrumentations.yml]: Do we need to adjust language or maybe add a comment on runtime dep?

File: _data/openzipkin_instrumentations.yml (hunk 1 above)
Comment (basvanbeek) [inline _data/openzipkin_instrumentations.yml]: updated to match rename

File: _data/openzipkin_instrumentations.yml (hunk 2 above)
Comment (adriancole) [inline _data/openzipkin_instrumentations.yml]: maybe comment that the library is written in java

File: pages/existing_instrumentations.md (hunk 3 above)
Comment (basvanbeek) [inline pages/existing_instrumentations.md]: Made it into this:

The following libraries are supported by the OpenZipkin authors and are hosted at the [OpenZipkin Github](https://github.com/openzipkin/) group. You can reach out to the authors at the [Zipkin Gitter](https://gitter.im/openzipkin/zipkin/) chat.

File: _data/openzipkin_instrumentations.yml (hunk 2 above)
Comment (basvanbeek) [inline _data/openzipkin_instrumentations.yml]: done

**Comment by basvanbeek** [comment]
Code comments have been addressed.

**Pull request merged**